README.md

```diff
@@ -99,6 +99,32 @@ You can have the tool create a machine-readable log file by specifying the `--lo
   -y, --yaml=<yamlFile>                  The path to the yaml file. (Not needed for the get commands)
 ```
 
+##### [HINT - DEFAULT VALUES FOR SOME PARAMS]
```

Comment:
I wouldn't call these default values, that's a bit misleading. Maybe adjust the title to point out it's about data in the YAML configs.